tests/test_executors.py

```diff
@@ -64,13 +63,25 @@ def getDataIds(self, field):
         return [quantum.dataId[field] for quantum in self.quanta]
 
 
+class QuantumMock:
+    def __init__(self, dataId):
+        self.dataId = dataId
+
+    def __eq__(self, other):
+        return self.dataId == other.dataId
+
+    def __hash__(self):
+        # dict.__eq__ is order-insensitive
+        return hash(sorted(kv for kv in self.dataId.items()))
```

Comment:
I think `sorted()` returns a `list` which is not hashable.

Reply:
Yes; fixed in #120. Sorry for the hasty merge!